### PR TITLE
OCaml 4.13.0 release

### PR DESCRIPTION
**Pull request by Octachron**
In preparation of the 4.13.0 (and 4.12.1) release tomorrow.

### CODE REVIEW

File: ocaml_version.ml [hunk 1]
```diff
@@ -160,7 +160,8 @@ module Releases = struct
   let v4_11 = v4_11_2
 
   let v4_12_0 = of_string_exn "4.12.0"
-  let v4_12 = v4_12_0
+  let v4_12_1 = of_string_exn "4.12.1"
+  let v4_12 = v4_12_1
 
   let v4_13_0 = of_string_exn "4.13.0"
```
Comment (dra27) [inline ocaml_version.ml:166]: Please could `v4_12_1` and `v4_13_0` go in the `.mli` as well

**Review by dra27 — COMMENTED** [review]
Thanks - ping me when the tags are pushed tomorrow and we'll set the base image builder updating with this as well (it vendors ocaml-version, so that can happen before the updated library is released)

**Comment by kit-ty-kate** [comment]
The opam-repository PR got merged. This is now good to go

**Comment by dra27** [comment]
Thanks!

File: ocaml_version.ml [hunk 2]
```diff
@@ -174,18 +175,18 @@ module Releases = struct
...
     v4_10_1; v4_10_2; v4_11_0; v4_11_1; v4_11_2;
-    v4_12_0; v4_13_0; v4_14_0 ]
+    v4_12_0; v4_12_1; v4_13_0; v4_14_0 ]
 
   let all = [ v4_00; v4_01; v4_02; v4_03; v4_04;
               v4_05; v4_06; v4_07; v4_08; v4_09;
               v4_10; v4_11; v4_12; v4_13; v4_14 ]
 
-  let recent = [ v4_02; v4_03; v4_04; v4_05; v4_06; v4_07; v4_08; v4_09; v4_10; v4_11; v4_12 ]
+  let recent = [ v4_02; v4_03; v4_04; v4_05; v4_06; v4_07; v4_08; v4_09; v4_10; v4_11; v4_12; v4_13 ]
```
[the remark this answers is not in the document]
Comment (dra27) [reply ocaml_version.ml:184]: Hopefully we'll get to reset it to:

```ocaml
let recent = [ v5_00 ]
```

relatively soon 😉

**Pull request merged**